themes/default/content/blog/review-stacks/index.md

```diff
@@ -0,0 +1,105 @@
+---
+title: "Review Stacks: Collaborate in the Cloud"
+date: 2023-06-14
+meta_desc: Review Stacks give you an automated cloud environment for every pull request so your team can ship faster, safer, and more collaboratively.
+meta_image: meta.png
+authors:
+    - evan-boyle
+    - pat-gavlin
+tags:
+    - cloud engineering
+    - pulumi deployments
+    - automation api
+    - deployments
+    - review stacks
+---
+
+Today we’re excited to announce Review Stacks -- dedicated cloud environments that get created automatically every time a pull request is opened, all powered by [Pulumi Deployments](/docs/pulumi-cloud/deployments). Open a pull request, and Pulumi Deployments will stand up a stack with your changes and the Pulumi GitHub App will add a PR comment with the outputs from your deployment. Merge the PR and Pulumi Deployments will destroy the stack and free up the associated resources. It has never been simpler to pick up an unfamiliar codebase, make changes to **both** application *and* infrastructure code, and share a live environment for review with your teammates.
+
+![Review Stacks Comment](comment.png)
+
+Ephemeral environments for front-end development and static websites have become commonplace in recent years, but infrastructure and complex systems have been unsupport up until this point. Review Stacks enable you to iterate on both application code changes **and** infrastructure code changes at the same time. Making changes to HTML, CSS, Next.js, Go APIs, Pulumi configuration, subnet configuration, databases, and load balancers are all possible with Review Stacks. No more fumbling with out-of-date READMEs and developer documentation. Just open a pull request and you can start testing changes against everything from simple static websites to API servers, microservices, data pipelines, Kubernetes clusters, and any other piece of infrastructure across Pulumi’s 100+ cloud providers.
+
+Review Stacks manage the full lifecycle of your cloud development environment including creating it when the PR is opened, updating it every time a new commit is pushed, and destroying and cleaning up all cloud resources when the pull request is merged or closed. They even integrate with GitHub checks so that you can gate your pull requests on successful Review Stack deployments.
```

Comment:
```suggestion
Review Stacks manage the full lifecycle of your cloud development environment including creation when a PR is opened, updates every time a new commit is pushed, and cleaning up all cloud resources when the pull request is merged or closed. They even integrate with GitHub checks so that you can gate your pull requests on successful Review Stack deployments.
```

Reply:
appreciate the suggestion, but the proposed sentence doesn't quite read right. Will keep as is for now.